tests/pgle_test.py

```diff
@@ -468,5 +468,54 @@ def check_if_cache_hit(event):
           self.assertLen(w, 1)
           self.assertIn("PERSISTENT CACHE WRITE with key jit_h-", str(w[0].message))
 
+  @parameterized.parameters([True, False])
+  def testAutoPgleWithCommandBuffers(self, enable_compilation_cache):
+    with (config.pgle_profiling_runs(1),
+          config.enable_compilation_cache(enable_compilation_cache),
+          config.enable_pgle(True),
+          tempfile.TemporaryDirectory() as dump_dir,
+          tempfile.TemporaryDirectory() as cache_dir):
+      if enable_compilation_cache:
```

Comment:
Please decorate this test case with `@jtu.thread_unsafe_test()`

(We have some test harnesses that run multiple test cases in parallel, and clearing the global compilation cache and expecting anything to be true about the cache directory is a very much thread-hostile behavior.)

Reply:
Done 👍